.sync.yml

```diff
@@ -7,11 +7,6 @@ Gemfile:
   extra:
     - gem: webmock
```

Comment:
That's master, but the CHANGELOG for 2.x and the published gems state 1.9.3 (https://github.com/bblimke/webmock/blob/master/CHANGELOG.md). I think that may become 3.x, so the version here could be raised to permit 2.x.